### DISCUSSION TITLE
KeyError: 'throat.hydraulic_conductance'

### DISCUSSION THREAD

**Discussion by yd66666666** [PMEAL/OpenPNM#2374] [Q&A]
Dear Professor, I am just a beginner, when I run this code, the following problem occurs. Thank you very much for your reply.
![image](https://user-images.githubusercontent.com/101027606/163926982-51b6f2ec-c38a-4e48-b38a-50233782553c.png)
![image](https://user-images.githubusercontent.com/101027606/163927116-91799260-bd1a-4e29-8ccb-d95e7e9f2ac1.png)

**Comment by jgostick** [comment]
It's because your hydraulic conductance was not calcuated....probably because some information was missing at a previous step.

**Reply by yd66666666** [reply]
Thank you very much and I'll reassemble it

**Comment by likeduck** [comment]
我遇到了相同的问题，请问您解决了吗？
I have encountered the same problem. Have you solved it ?
